docs/CI-CD/GithubActions.md

```diff
@@ -28,33 +30,65 @@ echo $He_SP_PWD_CICD
 
 ```
 
-## Add Service Principal Credentials to Github Secrets
+### Create a Docker hub User account and Create Personal Access Token
+
+ADD STEPS HERE
+
+### Set up Secrets in GitHub Action workflows
+
+[GitHub Secrets](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets) are encrypted and allow you to store sensitive information, such as access tokens, in your repository.
+
+You could use GitHub secrets to store your Azure Credentials, Publish profile of your Web app, container registry credentials or any such sensitive details which are required to automate your CI/CD workflows using GitHub Actions.
+
```

Comment:
I would add a sentence about not having to worry that these get checked into github as they are stored separately.